src/MPAGSCipher/CaesarCipher.cpp

```diff
@@ -1,38 +1,49 @@
-#include "RunCaesarCipher.hpp"
+#include "CaesarCipher.hpp"
+#include "CipherMode.hpp"
 
+#include <iostream>
 #include <string>
-#include <vector>
 
-std::string runCaesarCipher(const std::string& inputText, const std::size_t key,
-                            const bool encrypt)
-{
+CaesarCipher::CaesarCipher(const std::size_t cipherKey) {
+    key_ = cipherKey;
+}
```

Comment:
Better to use the member initialisation list:
```suggestion
CaesarCipher::CaesarCipher(const std::size_t cipherKey) : key_{cipherKey} {
}
```
Also, since you have the alphabet as a member variable (importantly declared before `key_`) you can do the truncation of the key at construction time:
```suggestion
CaesarCipher::CaesarCipher(const std::size_t cipherKey) : key_{cipherKey % alphabet_.size()} {
}
```